README.md

```diff
@@ -853,6 +899,14 @@ Component that represents a URL to an image, video or audio file. Used within ge
 ### new Media()
 
 
+| Param | Type | Description |
+| --- | --- | --- |
+| opts.url | <code>string</code> | Required |
+| opts.type | <code>string</code> | Required |
```

Comment:
Is this really needed? It seems like it is being repeated, if I look at the README. Please have a look at the screenshot below.
<img width="358" alt="image" src="https://user-images.githubusercontent.com/76547214/212244735-5b9d8246-8cb6-421b-be77-015b6de24598.png">

Reply:
it is auto generated, fixed in latest commit